editor/components/inserter/menu.js

```diff
@@ -201,9 +202,7 @@ export class InserterMenu extends Component {
 	renderCategories( visibleItemsByCategory ) {
 		if ( isEmpty( visibleItemsByCategory ) ) {
 			return (
-				<span className="editor-inserter__no-results">
```

Comment:
These two removed classNames are already present on the stylesheet. Should we remove them and use one of them to style the newly introduced `editor-inserter__no-blocks` className